### ISSUE TITLE
Dasboard Internal error on fresh install Ubuntu 20.04.2 LTS (jinja exception domains_types)

### ISSUE THREAD

**Issue by jbatest** [ail-project/ail-framework#110]
Hi. First of all congratz about your project, it's nice and usefull for the community ;)

I've a fresh installation over Ubuntu 20 and I've an 500 error (server internal) when accessing crawlers dashboard. ( /crawlers/dashboard). However, if i go to /crawler/settings I'm able to configure and get the splash_manager and the crawling working.

I've workarounded modifying the html files and quit the domain type search from the code. All the other features seems to work nice. However, it would be nice to fix the source of the problem.

This is the error taken from AIL.Flask Screen when accesing to /crawlers/dashboard. It seems to be a problem with "domains_types" variable but i don't know why.

**jinja2.exceptions.UndefinedError: 'domains_types' is undefined**
```
IP - - [01/Jun/2021 10:49:57] "GET /_stuff HTTP/1.1" 200 -
[2021-06-01 10:49:57,973] ERROR in app: Exception on /crawlers/dashboard [GET]
Traceback (most recent call last):
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/flask/app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/flask/app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/flask/app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/flask/app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "./modules/Role_Manager.py", line 98, in decorated_view
    return func(*args, **kwargs)
  File "/opt/ail-framework/var/www/blueprints/crawler_splash.py", line 64, in crawlers_dashboard
    return render_template("dashboard_splash_crawler.html", all_splash_crawler_status = all_splash_crawler_status,
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/flask/templating.py", line 147, in render_template
    return _render(
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/flask/templating.py", line 128, in _render
    rv = template.render(context)
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/opt/ail-framework/var/www/templates/crawler/crawler_splash/dashboard_splash_crawler.html", line 95, in top-level template code
    {% include 'domains/block_domains_name_search.html' %}
  File "/opt/ail-framework/var/www/templates/domains/block_domains_name_search.html", line 16, in top-level template code
    <input class="custom-control-input" type="checkbox" name="domain_onion_switch" value="" id="domain_onion_switch" {%if 'onion' in domains_types or not domains_types%}checked{%endif%}>
jinja2.exceptions.UndefinedError: 'domains_types' is undefined
IP - - [01/Jun/2021 10:49:57] "GET /crawlers/dashboard HTTP/1.1" 500 -
```
¿Can you please help guys? Thanks!

**Comment by Terrtia** [comment]
Hi @jbatest !
Thans for the report ! Fixed with 777eed3

Let me know if this solve your issue

**Comment by jbatest** [comment]
Hi @Terrtia ! Thank you for you support.

I've modified both files but it seems that there's still a "similar" problem this time with "block_languages_search.html"

¿Can you please check it? This is the full log of Flask Screen:
```
[2021-06-09 14:26:45,565] ERROR in app: Exception on /crawlers/dashboard [GET]
Traceback (most recent call last):
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/flask/app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/flask/app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/flask/app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/flask/app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "./modules/Role_Manager.py", line 98, in decorated_view
    return func(*args, **kwargs)
  File "/opt/ail-framework/var/www/blueprints/crawler_splash.py", line 64, in crawlers_dashboard
    return render_template("dashboard_splash_crawler.html", all_splash_crawler_status = all_splash_crawler_status,
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/flask/templating.py", line 147, in render_template
    return _render(
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/flask/templating.py", line 128, in _render
    rv = template.render(context)
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/opt/ail-framework/AILENV/lib/python3.8/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/opt/ail-framework/var/www/templates/crawler/crawler_splash/dashboard_splash_crawler.html", line 122, in top-level template code
    {% include 'domains/block_languages_search.html' %}
  File "/opt/ail-framework/var/www/templates/domains/block_languages_search.html", line 20, in top-level template code
    <input class="custom-control-input" type="checkbox" name="domain_onion_switch" value="" id="domain_onion_switch" {%if 'onion' in domains_types%}checked{%endif%}>
jinja2.exceptions.UndefinedError: 'domains_types' is undefined
```
Thank you in advance,

**Comment by Terrtia** [comment]
Thank you for the log !

Should be fixed with dfb8ea4

Let me know if you encounter another issue.

**Comment by jbatest** [comment]
Thank you so much Terrtia!! ;) Nice work, is working fine now! 👍
